src/mrfgen/mrfgen.py

```diff
@@ -2124,19 +2303,37 @@ def create_vrt(basename, empty_tile, epsg, xmin, ymin, xmax, ymax):
                          'Check stderr file: ',
                          gdal_translate_stderr_filename])
     log_sig_exit('ERROR', mssg, sigevent_url)
+
+
+def clean_mrf(data_filename): # cleans mrf files in place
```

Comment:
The helper functions are defined before the main flow of the script, so you may want to move this up there too to be consistent.

Reply:
Done